### ISSUE TITLE
Remove collections from advanced filters

### ISSUE THREAD

**Issue by thomaspiller** [unep-grid/map-x-mgl#660]
Collections should be removed from the advanced filters panel without being completely removed from MapX. They are always useful for sharing a subset of views with other users through a url.

**Comment by fxi** [comment]
I could rename "collections" to "tags", but I think removing those from the filters panel does not serve any rational purpose.  Sorry if I missed the rationale behind that.

As it's in "advanced filter", it's not visible, unless the user chose to display it. I don't understand why it should be removed.

I can off course remove all mapx functionalities, that's not the question. I just need a rational explanation, especially if it's labeled "priority 1".

Many software has a tagging system, which is used for filtering. If we look closely to this issue tracking system, there is one. It's useful. Github choose to keep it, there is a reason, even if some people don't know how to use it.

The "collections" were preferred to the sub-project system, because it was faster to implement. But then, I implemented sub-project, which did not trigger any reaction and this stayed in a stale state. Now we have also "groups" which is named "categories", in top of that. Those are not filterable. If we remove "collections", there is no more filtering options. Is that wise ?

**Comment by PierreLacroix** [comment]
I thought we agreed in a separate conversation that we would remove the collections from the advanced filter. As mentioned in this conversation, half of the views do not have collections. The ones that have collections are not WESR projects, and for them there is a high redundancy between the collection and either the view title, the project title, or the keywords. Ultimately we will move the collections into keywords.
Thanks a lot.

**Comment by fxi** [comment]
Sorry, I did not recall any agreement to be set on this. It's mentioned in that conversation that this action was not urgent and now it's a priority 1, so it is worth a little further discussion beforehand.

I thought I've made a point saying that a tagging system linked to a filtering system was useful, and that I could rename "collections" to "tags" or remove it partially or completely, but I did not agree. The argument that administrator who work on a specific project accepts tags that contain the same information as the content of title or description should not be the driver of this feature removal decision. If a specific project does not make use of all functionality is not a reason to remove this functionality. Furthermore, the argument of the redundancy does not stand as "titles" and "descriptions" are multilingual, and tags are language independent: they don't exist at the same level. 

Tagging is used everywhere in social media : people know how to use it. The term "collection" is poorly chosen as few people understand it. It would be wise to rename it to "tags" to make clearer what it does.

We could also let this as an option in the project configuration :

-[x] Remove "Filter by tags" panel in "advanced filter"

Or we could also make the element disappear if there is no tags to display.

Here is an example of the simple solution of renaming "collections" to "tags":

![mx_filter_by_tags](https://user-images.githubusercontent.com/1196833/109937505-4608fa00-7ccf-11eb-9a56-fc3237fa5267.gif)
